packages/react-vapor/src/components/table-hoc/TableHOCUtils.ts

```diff
@@ -60,6 +60,16 @@ const getComponentIdFromPredicateId = (predicateId: string) => predicateId.split
 
 const getPredicateId = (tableId: string, componentId: string) => `${tableId}${PREDICATE_SEPARATOR}${componentId}`;
 
+const getPredicateIds = (tableId: string, state: IReactVaporState): string[] =>
+    _.chain(state.listBoxes)
+        .filter((list: IListBoxState) => {
+            const startWithIdRegexp = new RegExp(`^${tableId}(.+)`);
+            return startWithIdRegexp.test(list.id);
+        })

```

Comment:
What do you think of: 

```typescript
.filter((list: IListBoxState) => new RegExp(`^${tableId}(.+)`).test(list.id))
```

You could also extract the function since it's also used in getTablePredicates